lms/djangoapps/discussion_api/views.py

```diff
@@ -123,113 +149,158 @@ class ThreadViewSet(_ViewMixin, DeveloperErrorViewMixin, ViewSet):
-            string will be returned.
+        * page: The (1-indexed) page to retrieve. Defaults to 1.
 
-        * order_by: Must be "last_activity_at", "comment_count", or
-            "vote_count". The key to sort the threads by. The default is
-            "last_activity_at".
+        * page_size: The number of items per page. Defaults to 10, with a
+          maximum of 100.
 
-        * order_direction: Must be "asc" or "desc". The direction in which to
-            sort the threads by. The default is "desc".
+        * text_search: A search string to match. Retrieves any thread with
+          content that matches the search string, including the bodies of
+          comments in the thread.
 
-        * following: If true, retrieve only threads the requesting user is
-            following
+        * topic_id: The ID of the discussion topic to retrieve threads for.
+          Multiple topic_id queries can be defined to retrieve threads from
+          multiple topics at once.
 
-        * view: "unread" for threads the requesting user has not read, or
-            "unanswered" for question threads with no marked answer. Only one
-            can be selected.
+        * view: "unread" to retrieve threads that the requesting user has not
+          read, or "unanswered" to retrieve question threads that do not have a
+          marked answer. Only one view can be specified in a request.
 
         The topic_id, text_search, and following parameters are mutually
-        exclusive (i.e. only one may be specified in a request)
+        exclusive. Only one can be specified in a request.
 
-    **POST Parameters**:
+    **POST Parameters**
 
-        * course_id (required): The course to create the thread in
+        * course_id (required): The course to create the thread in.
 
-        * topic_id (required): The topic to create the thread in
+        * following (optional): A Boolean that indicates whether the user will
+          follow the thread upon its creation. Defaults to false.
 
-        * type (required): The thread's type (either "question" or "discussion")
+        * raw_body (required): The text for the thread, without HTML markup.
 
-        * title (required): The thread's title
+        * title (required): The title for the thread.
 
-        * raw_body (required): The thread's raw body text
+        * topic_id (required): The discussion topic to create the thread in.
 
-        * following (optional): A boolean indicating whether the user should
-            follow the thread upon its creation; defaults to false
+        * type (required): Specifies either "question" or "discussion" as the
+          type of thread to create.
 
-    **PATCH Parameters**:
+    **PATCH Parameters**
 
         topic_id, type, title, and raw_body are accepted with the same meaning
-        as in a POST request
+        as in a POST request.
 
-    **GET Response Values**:
+        In addition, abuse_flagged, following, and voted are accepted.
```

Comment:
Should we note that the description of these parameters are listed in the PATCH response?